src/internal/modules/billing/routes/bill-run.js

```diff
@@ -198,7 +198,7 @@ const routes = {
   },
   getBillingBatchDeleteInvoice: {
     method: 'GET',
-    path: '/billing/batch/{batchId}/delete-invoice/{invoiceId}',
+    path: '/billing/batch/{batchId}/delete-invoice/{invoiceId}/org/{originalInvoiceId}/rebill/{rebillInvoiceId}',
```

Comment:
You should remove the last part of this route /org/{originalInvoiceId}/rebill/{rebillInvoiceId}
and add originalInvoiceId and rebillInvoiceId as optional query params this will allow you to use the same route to delete normal as well as rebilled invoices.